src/Data/Syntax.hs

```diff
@@ -140,10 +141,11 @@ instance Declarations1 Identifier where
 
 -- | An accessibility modifier, e.g. private, public, protected, etc.
 newtype AccessibilityModifier a = AccessibilityModifier { contents :: Text }
-  deriving newtype (Eq, Ord, Show)
-  deriving stock (Foldable, Functor, Generic1, Traversable)
-  deriving anyclass (Declarations1, Diffable, FreeVariables1, Hashable1, ToJSONFields1, NFData1)
-  deriving (Eq1, Show1, Ord1) via Generically AccessibilityModifier
+  deriving (Declarations1, Diffable, Foldable, FreeVariables1, Functor, Generic1, Hashable1, ToJSONFields1, Traversable)
+
+instance Eq1 AccessibilityModifier where liftEq = genericLiftEq
+instance Ord1 AccessibilityModifier where liftCompare = genericLiftCompare
+instance Show1 AccessibilityModifier where liftShowsPrec = genericLiftShowsPrec
```

Comment:
1. No more `Eq`, `Ord`, & `Show` instances.

2. It’s a significant amount less work to compile the `Eq1`, `Ord1`, & `Show1` instances when defined this way than when using `-XDerivingVia`, speeding up compiles a bit more.